students/cmg2249.md

```diff
@@ -5,20 +5,10 @@ Name: Carlos Gonzalez-Romero
 UNI: cmg2249
 
 Github: [link](https://github.com/Cargo1284)
+GitHub README: [link](https://github.com/Cargo1284/Cargo1284/blob/main/README.md)
 
 ## Project Proposal
-
-1. Fantasy baseball daily stats reports for your team
-2. College football webscraper that tells you information on the projected closest games
-3. Baseball pitcher matchup report
-4. Budget Tracker
-5. To do list
+- [FantasyBaseballTrends.md](../projects/python/FantasyBaseballTrends.md)
```

Comment:
doesn't match the path you've put the files in, did you look at the PR after you opened it to make sure it worked? IF I click on your links they don't work, which suggests you never checked
![tmp](https://user-images.githubusercontent.com/3105306/219467726-00cb82a1-81a6-403b-9299-1ab4d888075b.gif)